electrs-blockchain-provider/src/lib.rs

```diff
@@ -21,6 +20,7 @@ const MIN_FEERATE: u32 = 253;
 
 #[derive(Clone, Eq, Hash, PartialEq)]
 pub enum Target {
+    Minimum = 1008,
```

Comment:
Can't you just get rid of this enum like you did with a similar enum above?

Reply:
I had just applied a patch that was made to the master branch a while ago, but got rid of the enum now.

Reply:
Thanks!